pdq/wasm/Makefile

```diff
@@ -0,0 +1,31 @@
+.PHONY: all check-dependencies build test 
+.DEFAULT: all
+CWD := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+
+all: check-dependencies build test
+
+build: check-dependencies
+	@docker pull emscripten/emsdk:3.1.7
+	@docker run --rm -v $(CWD)/../..:/src emscripten/emsdk:3.1.7 emmake make wasm --directory=pdq/cpp
+	
+test:
+	@[ -f $(CWD)/../cpp/pdq-photo-hasher.js -a -f $(CWD)/../cpp/pdq-photo-hasher.wasm ] || (echo "Build the web assembly before running test targets"; /bin/false)
+	@(cd $(CWD); cp ../cpp/pdq-photo-hasher.* scripts)
+	@(cd $(CWD); npm install)
+	@(cd $(CWD)/test/WebAssemblySeleniumWebDriver/WebAssemblySeleniumWebDriver; dotnet build)
+	@(cd $(CWD); echo "Starting node server.."; nohup node server.js &)
+	(cd $(CWD)/test/WebAssemblySeleniumWebDriver/WebAssemblySeleniumWebDriver; dotnet run chrome pdqmd5 $(CWD)/test/TestData/PDQMD5TestData.csv http://localhost:9095)
+	@(kill -1 `ps -ef | grep 'node server.js' | grep -v grep | awk '{print $$2}'`)
+
+check-dependencies:	 
+	@node -v > /dev/null || (echo "Node is not installed. Exiting..."; /bin/false) 
+	@dotnet --version > /dev/null || (echo "dotnet Core 6.0 is not installed. Exiting..."; /bin/false) 
```

Comment:
dotnet is not strictly required for the build stage. Can we break up check-dependencies into check-build-dependencies and check-test-dependencies and use the correct ones as target dependencies in the various targets?